ts-rs/tests/semver.rs

```diff
@@ -0,0 +1,19 @@
+#![allow(deprecated)]
```

Comment:
Just curious, what is deprecated in this file?

Reply:
I just forgot to remove this line sorry